LiteFarmOrg/LiteFarm
# PLACEHOLDER: Refresh store with newly POSTed irrigation task

To Test,

1. create an irrigation task. make sure to checkbox everything that can be persisted
2. create another irrigation task by selecting the exact farm location.
3. check if the default values are persisted in the create irrigation task form
